pyworkflow/protocol/constants.py

```diff
@@ -65,7 +65,7 @@
 
 # Steps execution mode
 STEPS_SERIAL = 0      # Execute steps serially, some of the steps can be mpi programs
-STEPS_PARALLEL = 1    # Execute steps in parallel, through threads or mpi
```

Comment:
We might consider keeping this Constant for compatibility and avoiding errors in plugins if it is used. We can add a comment about its deprecation

Reply:
the constant is still there, I did not remove it